esmvalcore/preprocessor/__init__.py

```diff
@@ -563,20 +566,20 @@ def apply(self, step: str, debug: bool = False):
         )
 
     @property
-    def cubes(self):
+    def cubes(self) -> CubeList:
```

Comment:
This function returns a `list[Cube]` and not a `CubeList`. I would recommend annotating to match the current behaviour, but if you prefer you could also change the function so it returns a `CubeList`.

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2772/commits/62c1996a634373a158bcacb0d84c394a5073f557